app/assets/main/react_components/dashboard/climate_actions/manage_actions/kanban/card_components/Card.jsx

```diff
@@ -0,0 +1,137 @@
+  }, [sidebarCollapsed]);
+
+  const setStyleWithoutReordering = (style, snapshot) => {
+    /** Moving element from accepted column to achieved column */
+    if (!snapshot.isDragging) {
+      return {
+        userSelect: "none",
+        padding: 0,
+        minHeight: "auto",
+      };
+    }
+  };
+
+  const setStyleWithReordering = (style) => {
+    /** Moving element inside accepted column */
+    return {
+      userSelect: "none",
+      padding: 0,
+      minHeight: "auto",
+      ...style,
+    };
+  };
+
+  return (
+    <Draggable
+      key={item.id}
+      draggableId={item.id}
+      index={index}
+      isDragDisabled={item.status !== false}
+    >
+      {(provided, snapshot) => {
+        return (
+          <div
+            className={`rounded-lg h-20 space-y-3 mb-2 focus:outline-none relative justify-center ${
+              sidebarCollapsed
+                ? "d:w-24"
+                : "w-80 border border-gray-tint-2 mx-5"
+            }
+            ${item.expanded ? "h-auto" : "w-24"}`}
+            ref={provided.innerRef}
+            {...provided.draggableProps}
+            {...provided.dragHandleProps}
+            style={
+              item.status === false
+                ? setStyleWithReordering(provided.draggableProps.style)
+                : setStyleWithoutReordering(
+                    provided.draggableProps.style,
+                    snapshot
+                  )
+            }
+          >
+            {sidebarCollapsed ? (
+              <CardImage
+                img={isAchievement ? item.badge_image_url : item.image_url}
+                sidebarCollapsed={sidebarCollapsed}
+                isUserAction={isAchievement ? false : true}
+              ></CardImage>
+            ) : isAchievement ? (
+              <AchievementCard
+                categories={categories}
+                achievement={item}
+                sidebarCollapsed={sidebarCollapsed}
+                handleUncompleteAction={handleUncompleteAction}
+              ></AchievementCard>
+            ) : (
+              <UserActionCard
+                categoryColor={categoryColor}
+                userAction={item}
+                handleDelete={handleDelete}
+                handleCompleteAction={handleCompleteAction}
+                sidebarCollapsed={sidebarCollapsed}
+              ></UserActionCard>
+            )}
+
+            {!sidebarCollapsed && (
+              <div>
+                <div
+                  className=" h-14 w-full top-0 mt-6 absolute justify-center"
+                  onClick={() => handleExpanded(item, !item.expanded)}
+                >
+                  <button
+                    className={`fas right-0 focus:outline-none mr-4 absolute ${
+                      item.status === true ? "mt-1" : "mt-4"
+                    } ${item.expanded ? "fa-chevron-up" : "fa-chevron-down"} `}
+                    onClick={() => handleExpanded(item, !item.expanded)}
+                  ></button>
```

Comment:
If the whole item is "a button" maybe we should remove the button and just put the chevron in the div? The only reason for keeping the button right now should be for screen readers... can we rename the whole div to button?